rubocop.yml

```diff
@@ -50,6 +49,12 @@ Style/AccessorMethodName:
   Enabled: false
 Style/Alias:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
```

Comment:
Or should we revert that and just disable the block delimiters cop inside specs? I thought about that, but seemed a crude solution, and semantic blocks sounded better.